docs/release.md

```diff
@@ -0,0 +1,26 @@
+# Release Process
+This document describes how to cut a `cri` plugin release.
+
+## Step 1: Update containerd vendor
+Before cutting a release, update the containerd vendor to latest
+version in the corresponding branch, and make sure all tests in
+CI pass https://k8s-testgrid.appspot.com/sig-node-containerd.
+## Step 2: Cut the release
+Draft a new release in https://github.com/containerd/cri/releases.
+## Step 3: Update `cri` version in containerd
+Send a PR to containerd to update `cri` vendor version to the
+newly released version.
+## Step 4: Publish release tarball for Kubernetes
+Publish the release tarball `cri-containerd-${VERSION}.${OS}-${ARCH}.tar.gz`
+```shell
+# Checkout cri to the newly released version.
+git checkout ${RELEASE_VERSION}
+
+# Update containerd version in vendor.conf to the new release
```

Comment:
Should probably make this "Step 4: Iterate step 1 updating the version of containerd in vendor.conf"

And at some point need to tag the release either here or above..

Reply:
We've tagged release in step 2. We have to tag the release first, so that containerd can update to our tagged release.

Done